### PR TITLE
[QA] Aiakos acceptance test project and GET/UPLOAD test features

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@flopezag @jesuspg 

#### Description
This PR solves: *CLAUDIA-5752* of *CLAUDIA-5747: FIWARE.Story.Ops.Toolkit.Aiakos.API.UploadKeys*
- Aiakos Acceptance Test Project.
- Test features:
 * 'Get Support keys'.
 * 'Upload Support keys'.

#### Testing
Locally.

### CODE REVIEW

**Comment by flopezag** [comment]
Binary file not shown...?

File: test/acceptance/aiakos_client/support_api_client.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+AIAKOS_SUPPORT_BASE = "{" + API_ROOT_URL_ARG_NAME + "}/support"
+AIAKOS_SUPPORT_REGION = AIAKOS_SUPPORT_BASE + "/{region_name}"
+AIAKOS_SUPPORT_SSHKEY = AIAKOS_SUPPORT_BASE + "/{region_name}/sshkey"
+AIAKOS_SUPPORT_GPGKEY = AIAKOS_SUPPORT_BASE + "/{region_name}/gpgkey"
+
+
+class AiakosSupportApiClient(RestClient):
+
+    """
+    This class implements the a RestClient for the Support resource of the Aiakos API.
```
Comment (flopezag) [inline test/acceptance/aiakos_client/support_api_client.py]: the a ?

**Comment by jframos** [comment]
Ui, _.pyc_ files have been uploaded. We need I .gitignore file! ... sorry.

**Comment by flopezag** [comment]
LGTM

File: test/acceptance/README.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Top_
+
+
+Settings
+--------
+
+Before executing the acceptance tests, you will need configure the properties file. To execute acceptance test on the
+experimentation environment, you will have to configured the file `resources/settings.json` properly:
+
```
Comment (jesuspg) [inline test/acceptance/README.rst]: resources/settings.json or config/settings.json?

File: test/acceptance/README.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  You need a valid private key (*host_key*) to connect to the host by SSH.
+
+
+Configuration parameters (settings):
+
+- **protocol**: Aiakos API Protocol (HTTP/HTTPS)
+- **host**: Aiakos API host.
+- **port**: Aiakos API port.
+- **resource**: Aiakos API base resource (e.i: /v1)
+- **host_name**: Name of the host (to be used as part as a ssh connection).
```
Comment (jesuspg) [inline test/acceptance/README.rst]: parameter host_name don't exist in config/settings.json

File: test/acceptance/README.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+- **host_user**: User name of the host.
+- **host_password**: Password for the previous user.
+- **host_key**: RSA key for ssh connections instead of previous user/password.
```
Comment (jesuspg) [inline test/acceptance/README.rst]: host_key is not equals to host_private_key_location (in config/settings.json)

File: test/acceptance/README.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+Acceptance test preparation
+===========================
+
+Prerequisites
+-------------
+
+- Python 2.7 or newer (2.x) (https://www.python.org/downloads/).
+- pip (https://pypi.python.org/pypi/pip).
+- virtualenv (https://pypi.python.org/pypi/virtualenv).
+- Aiakos (https://pdihub.hi.inet/fiware/fiware-aiakos/).
```
Comment (jesuspg) [inline test/acceptance/README.rst]: change url to github

**Comment by jesuspg** [comment]
please pass pep8

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**